bench/wtperf/wtperf_run_py/wtperf_run.py

```diff
@@ -301,26 +304,46 @@ def main():
         with open(config.batch_file, "r") as file:
             batch_file_contents = json.load(file)
 
+    # Check for duplicate operations, and exit if duplicates are found
+    # First, construct a list of all operations, including potential duplicates
+    all_operations = []
+    if config.batch_file:
+        for content in batch_file_contents:
+            all_operations += content["operations"]
+    elif operations:
+        all_operations += operations
+    # Next, construct a new list with duplicates removed.
+    # Converting to a dict and back is a simple way of doing this.
+    all_operations_nodups = list(dict.fromkeys(all_operations))
```

Comment:
A (marginally) simpler implementation would be using a set here
`len(all_operations) != len(set(all_operations))`